src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -2208,7 +2203,8 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
     } else if (ceph_argparse_flag(args, i, "--pretty-format", (char *)NULL)) {
       cfg->pretty_format = true;
     } else if (ceph_argparse_flag(args, i, "--try-netlink", (char *)NULL)) {
-      cfg->try_netlink = true;
+      // netlink used by default. option not required anymore.
+      // allow backwards compatibility.
```

Comment:
```suggestion
      // accept for compatibility.
```

Reply:
And now that https://github.com/ceph/ceph/pull/55287 is merged, let's rebase and add a comment in `qa/workunits/rbd/rbd-nbd.sh` where `-o try-netlink` is kept on purpose:
```
# resize test
# also test that try-netlink option is accepted for compatibility
DEV=`_sudo rbd device -t nbd -o try-netlink map ${POOL}/${IMAGE}`